identity/app/views/profile/deletion/accountDeletion.scala.html

```diff
@@ -80,6 +79,24 @@ <h1 class="identity-title">Are you sure you want to delete your account?</h1>
 
         <form class="form" novalidate action="@idUrlBuilder.buildUrl("/delete", idRequest)" role="main" method="post">
             @views.html.helper.CSRF.formField
+
+            <div class="identity-section">
+                <p class="identity-section__text">
+                    <p><b>Do you mind telling us why you wish to delete your account?</b></p>
+
+                    @radioField(accountDeletionForm("reason"), List(
+                        "accident"      -> "I have created an account by accident",
+                        "password"      -> "I accidentally entered my password as the username",
+                        "emails"        -> "I want to stop receiving emails",
+                        "trolls"        -> "I have been trolled in the comments section",
+                        "comments"      -> "I no longer want to comment",
+                        "privacy"       -> "I am concerned about my privacy on the Internet",
```

Comment:
It seems the Guardian convention is to lowercase internet:
> the internet

.. although my preference would just be to go for:

> I am concerned about my privacy online

`</bikeshedding>`